request-options.js

```diff
@@ -1,17 +1,26 @@
 let config = require('./config/config');
+const fs = require('fs');
 
 function getRequestOptions(options) {
+    const {
+        request: { ca, cert, key, passphrase, rejectUnauthorized, proxy }
+    } = config;
+    
     return {
-        strictSSL: config.request.rejectUnauthorized,
-        json: true,
         headers: { 'ph-auth-token': options.token },
-        ca: config.request.ca,
-        proxy: config.request.proxy,
-        cert: config.request.cert,
-        passphrase: config.request.passphrase
+        ...(_configFieldIsValid(ca) && { ca: fs.readFileSync(ca) }),
```

Comment:
Taking a look at the usage of `getRequestOptions` and it appears to be invoked on every lookup request to the integration.  Assuming that's the case, won't this cause every call to `doLookup` to read the CA file off disk in a synchronous manner?

Reply:
That's correct.  Just added an external variable to 'cache' it to avoid repeating it.  Let me know if this could cause any issues I'm not aware of.